libraries/core-react/src/components/Popover/Popover.tokens.ts

```diff
@@ -24,61 +24,50 @@ const {
   elevation: { overlay: elevation },
 } = tokens
 
-type Popover = {
-  header: Typography
-  background: string
+type Popover = ComponentToken & {
   elevation: string
```

Comment:
Maybe we need that `boxShadow` property on `ComponentToken` 🙈 Suggest using `Paper` component in `Popover` nonetheless

Reply:
We're actually using `Paper` so elevation on `Popover` seem to be unused, will remove it 🔥